manifests/source.pp

```diff
@@ -18,7 +18,7 @@
   $key_content       = undef,
   $key_source        = undef,
   $trusted_source    = undef,
-  $notify_update     = undef,
+  $notify_update     = true,
```

Comment:
I think this is actually related to future parser and 3.7.2. Future parser's not always behaved correctly on 3.7 especially not before 3.7.3. If this doesn't show up with 3.8.x I'm inclined to leave this as is and count it as an FP bug on an older Puppet release, one I'm not super inclined to fix.